airstack-modules/modules/airstack/domain-name/domain-name.ts

```diff
@@ -643,6 +556,29 @@ export namespace domain {
   }
 
   // end of track functions and start of get or create and helper functions
+  /**
+   * @dev this function creates an air extra entity
+   * @param name air extra name
+   * @param value air extra value
+   * @param domainId air domain id
```

Comment:
You can rename it as id and move the function in to common.

Reply:
cannot be done as all air extras dont have a domain id linked to them, social module has user id, so the air extra entity schema differs